project_event/views/project_task_view.xml

```diff
@@ -86,7 +112,16 @@
                             <field name="notes"/>
                         </page>
                         <page string="Technical Sheet">
-                            <field name="spectators"/>
+                        <group>
+                            <field name="spectators" style='max-width: 900px; width: 100px'/>
+                            <field name="table" widget="widget_table" options="{
+                            'field_name': 'child_ids',
+                            'datatable_params':{'searching': False,
+                            'paging': False,
+                            'info': False},
+                            'fields': ['name','date_start', 'date_end', 'real_date_start', 'real_date_end', 'total_time'],
+                            'headers': ['Name','Date Start', 'Date End', 'Read Date Start', 'Real Date End', 'Total Time']}" />
```

Comment:
are these headers translatable?

Reply:
No, this feature is missing for the moment

Reply:
@rimbendhaou @eilst  Hope Luis is taking care in his branch.